layout/pulse/layer-card/actions.js

```diff
@@ -15,7 +15,7 @@ export const loadDatasetData = createThunkAction('layer-card/loadDatasetData', p
         language: params.locale || 'en'
       });
 
-      datasetService.fetchData().then((data) => {
+      datasetService.fetchData('metadata').then((data) => {
```

Comment:
I'm trying to avoid using services due to their duplication and because I think that is easier to manage changes on each call

Reply:
👍